test/unittests/commands/xmlupload/make_rdf_graph/test_make_values.py

```diff
@@ -351,74 +310,26 @@ def test_segment_of_audio(self, lookups: Lookups) -> None:
         assert value == RES_ONE_URI
 
 
-class TestMakeOnePropGraphRaises:
-    def test_permissions(self, lookups: Lookups) -> None:
```

Comment:
It is not possible for some error to occur anymore, because permissions resolving etc. would have been noticed earlier.